exporters/otlp/otlptrace/otlptracehttp/client_test.go

```diff
@@ -219,7 +218,8 @@ func TestTimeout(t *testing.T) {
 		assert.NoError(t, exporter.Shutdown(ctx))
 	}()
 	err = exporter.ExportSpans(ctx, otlptracetest.SingleReadOnlySpan())
-	assert.Equalf(t, true, os.IsTimeout(err), "expected timeout error, got: %v", err)
```

Comment:
Can we also keep this assertion that we can unwrap error?

Reply:
It's difficult to preserve the behavior of `os.IsTimeout()`, actually, when you wrap an error. I think we can do is create a struct to wrap errors and prefix in the `Error()` method, we have to avoid `fmt.Errorf()`.